tests/tests.py

```diff
@@ -526,53 +560,25 @@ def test_res_comp_with_no_podpolicies(self):
     def test_appstatus_plugin(self):
         kubeplus_home = os.getenv("KUBEPLUS_HOME")
         provider = kubeplus_home + '/kubeplus-saas-provider.json'
-        
-        def cleanup():
-            cmd = "kubectl delete -f ../examples/multitenancy/hello-world/hs1.yaml --kubeconfig=%s" % provider
-            TestKubePlus.run_command(cmd)
-            cmd = "kubectl delete -f ../examples/multitenancy/hello-world/hello-world-service-composition-localchart.yaml --kubeconfig=%s" % provider
-            TestKubePlus.run_command(cmd)
-
-        if not TestKubePlus._is_kubeplus_running():
-            print("KubePlus is not running. Deploy KubePlus and then run tests")
-            sys.exit(0)
 
-        if os.getenv("KUBEPLUS_HOME") == '':
-            print("Skipping test as KUBEPLUS_HOME is not set.")
-            return
-
-        # register HelloWorldService API
-        cmd = "kubectl create -f ../examples/multitenancy/hello-world/hello-world-service-composition-localchart.yaml --kubeconfig=%s" % provider
-        TestKubePlus.run_command(cmd)
+        self.setup_example_hello_world(provider=provider)
```

Comment:
Don't need 'provider=provider', since provider is not a keyword argument in line 50.
Just pass provider.